frontend/src/components/Shared/PrivateRoute.js

```diff
@@ -0,0 +1,20 @@
+import React from 'react';
+import { Route, Redirect } from 'react-router';
+import routePaths from '../../constants/routePaths';
+
+export const PrivateRoute = ({ component: Component, ...rest }) => (
```

Comment:
you are exporting this two times. One here and the `default` at the end of the file

Reply:
fixed